sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/code.py

```diff
@@ -34,18 +34,7 @@ def is_file_excluded(self, file_path: Union[str, Path]) -> bool:
         for ignore_file in self._extra_ignore_list:
             if ignore_file.is_file_excluded(file_path):
                 return True
-        return super(InternalComponentIgnoreFile, self).is_file_excluded(file_path)
-
-    def merge(self, other_path: Path) -> "InternalComponentIgnoreFile":
-        """Merge ignore list from another InternalComponentIgnoreFile object."""
-        if other_path.is_file():
-            return self
-        return InternalComponentIgnoreFile(other_path, extra_ignore_list=self._extra_ignore_list + [self])
-
-    def rebase(self, directory_path: Union[str, Path]) -> "InternalComponentIgnoreFile":
-        """Rebase the ignore file to a new directory."""
-        self._base_path = directory_path
-        return self
+        return IgnoreFile.is_file_excluded(self, file_path)
```

Comment:
Why this? And given we have moved some logic to its super class, this function should be possible to simplify?

Reply:
Because ComponentIgnoreFile also have is_file_excluded for v2 command component, and we don't expect internalComponentIgnoreFile  call it here.

Reply:
I will upddate logic here and remove internalComponentIgnoreFile class